### PR TITLE
Add is_test option

### PR DESCRIPTION
**Pull request by mogusbi**
This pull request adds the `is_test` option when checking for spam.

Library and readme have been updated. The gitignore file has also been updated because I use webstorm as my IDE and it adds a load of artefacts in a .idea folder that you really don't want added to source control

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,7 @@ client.checkSpam({
   comment_author_email : 'john.smith@gmail.com',
   comment_author_url : 'http://johnsblog.com',
   comment_content : 'Very nice blog! Check out mine!'
```
Comment (chrisfosterelli) [inline README.md]: For this example code to be valid, we need a comma at the end of this line

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "akismet-api",
-  "version": "2.1.1",
+  "version": "2.1.2",
```
Comment (chrisfosterelli) [inline package.json]: Are you able to leave this at 2.1.1? I think this should be 2.2.0 to be in line with semantic versioning, and I'll need to update the CHANGELOG so I'll likely just do a separate PR which bumps the version before publish :)

**Comment by mogusbi** [comment]
I've reverted that version number and amended the readme

**Comment by chrisfosterelli** [comment]
Thanks again! I'd be happy to do a passthrough for whatever options the API supports, so feel free to make an issue or PR if you need those fields.

**Comment by chrisfosterelli** [comment]
Published in 2.2.0

**Pull request merged**